extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -216,6 +220,9 @@
       </div>
       <div class="input-tag" ng-if="$ctrl.objectIsSelected">
         Layer number:{{ $ctrl.layerNum }}
+        <i class="material-icons layerInfo" tooltip-placement="bottom" uib-tooltip="This denotes the layer in which the shape is present. Layer 1 is the bottom layer.">
```

Comment:
We don't use camerCase words for class names. Please rename.

Reply:
Done.